src/ProductBundle/Command/GenerateProductCommand.php

```diff
@@ -131,7 +131,7 @@ protected function execute(InputInterface $input, OutputInterface $output): void
             sprintf('%s/Resources/views/Entity', $bundle_dir),
         ]);
 
-        $product = $input->getArgument('product');
+        $product = ucfirst($input->getArgument('product'));
```

Comment:
Is this the only place, where input argument must be modified? I found 7 places in `execute`, where `$input->getArgument('product')` is called